api/views.py

```diff
@@ -74,14 +75,17 @@ def added(request):
 def add_site(request):
     """Handle incoming POST requests to add new sites to the list.
 
-    For sites submitted via the extension.
+    Creates a new SiteRequest object for sites submitted via the extension.
 
-    Returns JSON response.
+    Returns JSON response back to the extension.
 
     """
 
     add_site = request.POST['site']
 
+    site_request = SiteRequest(website=add_site)
```

Comment:
Disregard this. Apparently it will not error if the url is invalid. Hmm. We may want to validate ourselves in this case. But not too important.

If interested. https://docs.djangoproject.com/en/1.8/ref/validators/#urlvalidator